plugin.video.ted.talks/resources/lib/menu_util_test.py

```diff
@@ -0,0 +1,13 @@
+import unittest
```

Comment:
It's great that you have tests, but those are not required for running the add-on in Kodi. So please drop test related files from the PR.